pages/bookmarks.html

```diff
@@ -1,12 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../templates/css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
     <title>Quiz App</title>
 </head>
+
 <body>
-    
+    <header class="ts-center">
+        <h1>BOOKMARKS</h1>
+    </header>
+
+    <main>
+
+        <div class="container">
+            <section class="cards">
+                <button><i class="bookmark fa-solid fa-bookmark fa-2x"></i></button>
+                <h2 class="title_stlye">QUESTION:</h2>
+                <article class="ts-font-small question_box">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas, consequuntur dolores!</p>
+                </article>
+                <button class="button_show_answer" type="button" onclick="">SHOW ANSWER</button>
```

Comment:
We don't merge code we currently do not need.
```suggestion
                <button class="button_show_answer" type="button">SHOW ANSWER</button>
```